main.css

```diff
@@ -41,10 +41,36 @@ td{
 }
 
 .topbar{
+	display: flex;
+	flex-wrap: nowrap;
+	justify-content: space-evenly;
+	align-items:stretch;
+	width: 99%;
 	position: fixed;
-	background-color: #E5D9B9;
+	background-color: #FBEFD5;
 	padding:0.5em;
 	top: 0; /* hug the top of the window instead of having a buffer */
+	left: 0;
+}
+
+.topbarSection{
+	text-decoration: none;
+	flex-grow:5;
+	text-align: center;
+	background: #EBDFC5;
```

Comment:
add in max-height: 1em so only the moused over element extends down, not the entire topbar. this makes mousing over less obnoxious.
This breaks on mobile but mouseover already is kinda sketch on mobile so I can live with it until we further optimize the mobile menuing.